SDM/SDM/settings/base.py

```diff
@@ -116,9 +117,8 @@
 
 STATIC_URL = 'static/'
 
-STATICFILES_DIRS = [
-    BASE_DIR / 'static'
-]
+STATICFILES_DIRS = [os.path.join(BASE_DIR, 'static') ]
```

Comment:
What's the difference?

Reply:
They do the same but its advisable to use os.path.join incase of static issues in future when deploying.